src/sidebar/store/create-store.js

```diff
@@ -93,6 +91,74 @@ import { createReducer, bindSelectors } from './util';
  *   SelectorMethods<RootSelectors>} Store
  */
 
+/**
+ * Create a Redux reducer from a store module's reducer map.
+ *
+ * @template State
+ * @param {ReducerMap<State>} reducers
+ */
+function createReducer(reducers) {
```

Comment:
There are no changes to this function, compared to the version that was in `util.js`, except from the addition of types.

Reply:
Comprehension question: We end up generating one reducer per store module/"namespace" (versus one reducer per entry in the `ReducerMap`)?

Reply:
This is correct. We end up with an object like `{ moduleA: (state, action) => { ... }, moduleB: (state, action) => { ... } }` which we then pass to Redux's [`combineReducers` function](https://redux.js.org/api/combinereducers) that combines this into a single mega-reducer that is then passed to [`createStore`](https://redux.js.org/api/createstore).